client/slack-api-client/src/main/kotlin/io/olaph/slack/client/group/respond/RespondMethodGroup.kt

```diff
@@ -0,0 +1,11 @@
+package io.olaph.slack.client.group.respond
+
+interface RespondMethodGroup {
+
+    /**
+     * sends an ephemeral response to the response-url
+     */
+    fun ephemeral(responseUrl: String): RespondEphemeralMethod
```

Comment:
this should mot be called ephemeral since we also can respond with non ephemeral messages